config/jobs/kubernetes/sig-testing/apidiff.yaml

```diff
@@ -43,7 +43,7 @@ presubmits:
         # We get that from "git merge-base".
         #
         # -b can be used more than once.
-        - "./hack/apidiff.sh -r $(git merge-base ${PULL_BASE_SHA} ${PULL_PULL_SHA}) -t ${PULL_PULL_SHA} -b /workspace/sigs.k8s.io/controller-runtime"
+        - "./hack/apidiff.sh -r $(git merge-base ${PULL_BASE_SHA} ${PULL_PULL_SHA}) -t ${PULL_PULL_SHA} -b /home/prow/go/src/sigs.k8s.io/controller-runtimer"
```

Comment:
I used GOPATH

https://github.com/kubernetes/test-infra/blob/575c0177404a91fdb87622db41af6e298f1c5976/config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml#L44

Reply:
It seems comes from here 

https://github.com/kubernetes/test-infra/blob/575c0177404a91fdb87622db41af6e298f1c5976/images/kubekins-e2e/Dockerfile#L25-L27

Reply:
try with 
```suggestion
        - "./hack/apidiff.sh -r $(git merge-base ${PULL_BASE_SHA} ${PULL_PULL_SHA}) -t ${PULL_PULL_SHA} -b ${GOPATH}/src/sigs.k8s.io/controller-runtimer"
```

Reply:
That seems a bit better than using some hard-coded path. Changed.